eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -58,222 +54,736 @@ parameters:
-      vmrBranch: ${{ variables.VmrBranch }}
-      architecture: x64
-      pool: ${{ parameters.pool_Linux }}
-      container: ${{ parameters.centOSStream8Container }}
-      buildFromArchive: false            # 🚫
-      enablePoison: false                # 🚫
-      excludeOmniSharpTests: true        # ✅
-      runOnline: true                    # ✅
-      useMonoRuntime: false              # 🚫
-      withPreviousSDK: false             # 🚫
-
-  - template: ../jobs/vmr-build.yml@self
-    parameters:
-      # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
-      buildName: CentOSStream8_Offline_MsftSdk
-      isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
-      vmrBranch: ${{ variables.VmrBranch }}
-      architecture: x64
-      pool: ${{ parameters.pool_Linux }}
-      container: ${{ parameters.centOSStream8Container }}
-      buildFromArchive: true             # ✅
-      enablePoison: false                # 🚫
-      excludeOmniSharpTests: true        # ✅
-      runOnline: false                   # 🚫
-      useMonoRuntime: false              # 🚫
-      withPreviousSDK: false             # 🚫
-
-  - ${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
-
-    # CI - Stage 1 x64 legs ------------------------------------
-
-    - template: ../jobs/vmr-build.yml@self
+- ${{ if parameters.isSourceOnlyBuild }}:
+  - stage: VMR_SourceOnly_Build
+    displayName: VMR Source-Only Build
+    dependsOn: []
+    variables:
+    - template: ../variables/vmr-build.yml
       parameters:
-        # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
-        buildName: Alpine317_Offline_MsftSdk
-        isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
-        vmrBranch: ${{ variables.VmrBranch }}
-        architecture: x64
-        pool: ${{ parameters.pool_Linux }}
-        container: ${{ parameters.alpine317Container }}
-        buildFromArchive: false            # ✅
-        enablePoison: false                # 🚫
-        excludeOmniSharpTests: true        # ✅
-        runOnline: false                   # 🚫
-        useMonoRuntime: false              # 🚫
-        withPreviousSDK: false             # 🚫
+        vmrBranch: ${{ parameters.vmrBranch }}
+
+    jobs:
 
-    - template: ../jobs/vmr-build.yml@self
+    ### Jobs for ultralite builds ###
+    - template: ../jobs/vmr-build.yml
       parameters:
         # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
-        buildName: CentOSStream8_Online_PreviousSourceBuiltSdk
+        buildName: CentOSStream9_Online_MsftSdk
         isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
         vmrBranch: ${{ variables.VmrBranch }}
         architecture: x64
         pool: ${{ parameters.pool_Linux }}
-        container: ${{ parameters.centOSStream8Container }}
+        container: ${{ variables.CentOSStream9Container }}
```

Comment:
I am not seeing where these variables are coming from?

Reply:
This must be a bad merge with 1ES template. I'll take a look.